src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -425,26 +413,14 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
   }, [onNext])
 
   const missingBalanceForGasCryptoPrecision = useMemo(() => {
-    // Token withdraws aren't dust sends, they're actual contract calls
-    // Hence, the balance required for them is denominated in the native fee asset
-    if (isTokenWithdraw) {
-      return fromBaseUnit(
-        bnOrZero(feeAssetBalanceCryptoBaseUnit)
-          .minus(bnOrZero(state?.withdraw.estimatedGasCryptoBaseUnit))
-          .times(-1),
-        feeAsset.precision,
-      )
-    }
     return fromBaseUnit(
-      bnOrZero(assetBalanceBaseUnit)
+      bnOrZero(feeAssetBalanceCryptoBaseUnit)
```

Comment:
Nice cleanup 🐐 those two branches were literally the same assuming we were speaking fee asset